ui/tests/integration/components/recovery/page/snapshots-test.js

```diff
@@ -0,0 +1,122 @@
+  });
+
+  test('it displays empty state in non root namespace', async function (assert) {
+    this.nsService = this.owner.lookup('service:namespace');
+    this.nsService.path = 'parent1/child1';
+    this.server.get('/sys/internal/ui/namespaces', () => {
+      return { data: { keys: ['parent1/', 'parent1/child1/'] } };
+    });
+
+    await render(hbs`<Recovery::Page::Snapshots @model={{this.model}}/>`);
+
+    assert
+      .dom(GENERAL.emptyStateTitle)
+      .hasText('Snapshot upload is restricted', 'non root namespace empty state title renders');
+    assert
+      .dom(GENERAL.emptyStateMessage)
+      .hasText(
+        'Snapshot uploading is only available in root namespace. Please navigate to root and upload your snapshot. ',
+        'non root empty state message renders'
+      );
+    assert
+      .dom(GENERAL.emptyStateActions)
+      .hasText('Take me to root namespace', 'non root empty state action renders');
+
+    this.nsService.path = '';
+  });
+
+  test('it displays empty state when user cannot load snapshot', async function (assert) {
+    await render(hbs`<Recovery::Page::Snapshots @model={{this.model}}/>`);
+    assert.dom(GENERAL.emptyStateTitle).hasText('No snapshot available', 'empty state title renders');
+    assert
+      .dom(GENERAL.emptyStateMessage)
+      .hasText(
+        'Ready to restore secrets? Please contact your admin to either upload a snapshot or grant you uploading permissions to get started.',
+        'empty state message renders'
+      );
+    assert
+      .dom(GENERAL.emptyStateActions)
+      .hasText('Learn more about Secrets Recovery', 'empty state action renders');
+  });
+
+  test('it displays empty state when user can load snapshot', async function (assert) {
+    this.model.canLoadSnapshot = true;
+    await render(hbs`<Recovery::Page::Snapshots @model={{this.model}}/>`);
+    assert
+      .dom(GENERAL.emptyStateTitle)
+      .hasText('Upload a snapshot to get started', 'empty state title renders');
+    assert
+      .dom(GENERAL.emptyStateMessage)
+      .hasText(
+        'Secrets Recovery allows you to restore accidentally deleted or lost secrets from a snapshot. The snapshots can be provided via upload or loaded from external storage.',
+        'empty state message renders'
+      );
+    assert.dom(GENERAL.emptyStateActions).hasText('Upload snapshot', 'empty state action renders');
+  });
+
+  test('it displays loaded snapshot card', async function (assert) {
+    const expiryDate = dateFormat(
+      [new Date('2023-09-20T10:51:53.961861096-04:00'), 'MMMM do yyyy, h:mm:ss a'],
+      {}
+    );
+
+    this.model = {
+      snapshot_id: '6ecc06a9-3592-26f9-40cb-8d1b511890a6',
+      status: 'ready',
+      expires_at: expiryDate,
+    };
+
+    await render(hbs`<Recovery::Page::Snapshots::SnapshotManage @model={{this.model}}/>`);
+
+    assert.dom(SELECTORS.badge('status')).hasText('ready', 'status badge renders');
+  });
```

Comment:
This test file is for the `await render(hbs`<Recovery::Page::Snapshots @model={{this.model}}/>`);` component so this belongs in a test file for the SnapshotsManage component. 

That's fine if you want to leave for now and we can move it over when we finish building out that view, but generally it's good practice to only test one component per module/test file